crates/bevy_pbr/src/cluster/mod.rs

```diff
@@ -151,6 +151,10 @@ pub struct GpuClusterableObject {
     pub(crate) shadow_depth_bias: f32,
     pub(crate) shadow_normal_bias: f32,
     pub(crate) spot_light_tan_angle: f32,
+    pub(crate) soft_shadow_size: f32,
+    pub(crate) shadow_map_near_z: f32,
+    pub(crate) pad_a: f32,
+    pub(crate) pad_b: f32,
```

Comment:
Nit: Is this intended to make the structure's size a multiple of 4 x f32? If so a comment and maybe a test might be good, assuming it's important and relied upon elsewhere. Maybe something like this would also make the intention clearer and more maintainable:
```suggestion
    pub(crate) _reserved: [f32; 2],
```

The test could be as simple as this:
```rust
#[cfg(test)]
const_assert_eq!(size_of::<GpuClusterableObject>(), size_of::<Vec4>());
```

Reply:
A test isn't that necessary for that. Things break pretty quickly if the padding is wrong anyway. We have padding in a bunch of places for rendering code because of some webgl limitations.

Reply:
Cool, no problem if it’s convention - although should it be commented all the same?

Reply:
For me, just the fact that it's called `pad_x` is enough documentation.

Reply:
Righto, again no problem if it's an established convention in the codebase.

I was initially unsure whether this was perhaps something like padding from the edge of the shadow when sampling to avoid boundary conditions. It required me to check it wasn't used to understand its purpose and intention, but if I was adding a new field I may not have checked so thoroughly and just added new fields rather than removing these.

Again, it's a nit, and not a huge issue. I just saw it as something that could be made clearer. My intention if it's clearer is that more people can get across these changes, review, and contribute. Being knowledgable about graphics doesn't mean you're knowledgable about conventions in this codebase, or rust, etc. My understanding of Rust convention, for example, is that fields like this would have an underscore prefix `_pad_a` to indicate they're intentionally unused.